### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Rolv-Apneseth**
Add badges to the readme, along with some other minor changes. As I mentioned in #3, I used the [cargo-rdme](https://github.com/orium/cargo-rdme) tool so that also adds in 2 commented lines. Included also a section about adding this crate as a dependency, as in my opinion, even though it's obvious, it should have one considering the instructions start with creating a new project.

Also added a license file so that I could link to one in the badge but maybe that's unnecessary?

Let me know if this is alright or if you would like any changes.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015 The cargo-readme Developers
```
Comment (SabrinaJewson) [inline LICENSE]: Was this left in by accident? The cargo-readme devs definitely don’t have copyright over this code 😅 
I don’t really think we need a license file, it’s in the Cargo.toml that it’s MIT-licensed and that should be clear enough.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
-# rofi-mode
+# rofi-mode &middot; [![crates.io](https://img.shields.io/crates/v/cargo-rdme.svg)](https://crates.io/crates/cargo-rdme) [![License](https://img.shields.io/badge/License-MIT-green.svg)](https://github.com/SabrinaJewson/rofi-mode.rs/blob/main/LICENSE)
```
Comment (SabrinaJewson) [inline README.md]: Can’t we just have the badges in both the README and crate docs, but put them beneath the tagline? I think that’d be simpler as well as more useful (and we wouldn’t need cargo-rdme).

**Review by SabrinaJewson — COMMENTED** [review]
Thanks for making this!

File: LICENSE (hunk 1 above)
Comment (Rolv-Apneseth) [reply LICENSE]: Wow my bad, just copied the first MIT license I saw. Something I should put in place of that line or just remove it?
Edit: clearly my head is all over the place this weekend, can't read apparently. Yes I'll remove the file.

File: README.md (hunk 2 above)
Comment (Rolv-Apneseth) [reply README.md]: Yeah makes sense, wasn't sure if it was ok to put stuff like that in `lib.rs` (no experience with this apologies)

**Comment by Rolv-Apneseth** [comment]
Is that alright? Used `cargo-readme` and removed the license file.

**Comment by SabrinaJewson** [comment]
Thanks, looks good!

**Pull request merged**